test/test_corpus.cc

```diff
@@ -569,8 +569,8 @@ TEST_P(WhitequarkParserTest, PerPhaseTest) { // NOLINT
         {
             core::UnfreezeNameTable nameTableAccess(gs); // enters original strings
 
-            nodes = parser::Parser::run(gs, file);
-            TEST_COUT << "PARSED NODE" << nodes->nodeName() << "\n";
+            vector<string> locals = {"foo", "bar", "baz"};
+            nodes = parser::Parser::runWithLocals(gs, file, locals);
```

Comment:
Could you give more context to this change?
I'd find it super surprising that the commonly used function names have already been used for local variables.

Reply:
basically `whitequark/parser` introduces a few local variables before running each test: https://github.com/whitequark/parser/blob/master/test/test_parser.rb#L16-L18. It simplifies a lot of common use cases when you want to parse something that uses lvar without declaring it explicitly